_changes/2018-04-16.md

```diff
@@ -0,0 +1,8 @@
+---
+date: 2018-04-16
+---
+
+**Interop**
+
+ * Change: Reordered items under the [Interop guide for Java (for Kotlin consumption)](interop.html#java-for-kotlin-consumption) to roughly reflect their relative priority.
```

Comment:
I rank hard keywords above nullability so I'm not sure we want to communicate this as being a priority ordering.

Reply:
I moved hard keywords to be the first item. The priority is really just implicit. We don't say they are in priority order in the doc.